cmake/utils.cmake

```diff
@@ -4,6 +4,21 @@
 
 include(CMakeParseArguments)
```

Comment:
I don't think that file is necessary and can be removed. At least I do not see macro usage.